apps/ocean-api/__tests__/modules/PlaygroundModule.test.ts

```diff
@@ -0,0 +1,48 @@
+import { OceanApiTesting } from '../../testing/OceanApiTesting'
+import waitForExpect from 'wait-for-expect'
+
+describe('regtest', () => {
+  const apiTesting = OceanApiTesting.create()
+
+  beforeEach(async () => {
+    apiTesting.playgroundEnable(false)
+    await apiTesting.start()
+  })
+
+  afterEach(async () => {
+    await apiTesting.stop()
+  })
+
+  it('block count should not increment', async () => {
+    const initial = await apiTesting.rpc.blockchain.getBlockCount()
+
+    await new Promise((resolve) => {
+      setTimeout(_ => resolve(0), 5000)
+    })
+
+    const next = await apiTesting.rpc.blockchain.getBlockCount()
+    expect(next).toStrictEqual(initial)
+  })
+})
+
+describe('playground', () => {
+  const apiTesting = OceanApiTesting.create()
+
+  beforeEach(async () => {
+    apiTesting.playgroundEnable(true)
+    await apiTesting.start()
+  })
+
+  afterEach(async () => {
+    await apiTesting.stop()
+  })
+
+  it('block count should increment', async () => {
```

Comment:
`nit:` Following up on the previous statement
```
it('should increment block count')
```

Reply:
```suggestion
  it('should increment block count', async () => {
```